core/workspace_comment.js

```diff
@@ -95,6 +95,12 @@ Blockly.WorkspaceComment = function(workspace, content, height, width, opt_id) {
    */
   this.movable_ = true;
 
+  /**
+   * @type {boolean}
+   * @private
+   */
+  this.readOnly_ = false;
```

Comment:
Instead of calling this readOnly_, for consistency with a block, let's match the name here with the equivalent property ``editable_``.